src/controllers/user.controller.js

```diff
@@ -91,6 +91,7 @@ class userController {
     if (updaUser.status === 200) {
```

Comment:
@mujohn26 , this variable seems to be misspelled, kindly update this to a more intuitive name